cmd/namespace_client.go

```diff
@@ -247,13 +237,21 @@ func init() {
 	deleteCmd.Flags().StringVar(&prefix, "prefix", "", "prefix for delete namespace")
 
 	namespaceCmd.PersistentFlags().String("namespace-url", "", "Endpoint for the namespace registry")
-	err := viper.BindPFlag("NamespaceURL", namespaceCmd.PersistentFlags().Lookup("namespace-url"))
-	if err != nil {
+	// TODO: I don't think this flag is bound to any viper config name we have.
+	// Should instead be Federation.NamespaceURL
+	if err := viper.BindPFlag("NamespaceURL", namespaceCmd.PersistentFlags().Lookup("namespace-url")); err != nil {
```

Comment:
Absolutely, this one slipped past us when we refactored all of the viper variables. In fact, I'd go so far as to say we should change all of them to `Federation.RegistryUrl`, because I think that's a lot more clear, but we might be locked in because of our concerns about backwards compatibility... For now, can you make the change to `Federation.NamespaceUrl`?

Reply:
I will post renaming to `Federation.RegistryUrl` as a separate issue and @bbockelm might have a better idea of whether this will have compatibility issue for clients that's currently running as `stashcp` or `osdf` as that's the only place where I think we have people actually using them and will potentially require an update in the future.

Reply:
Ref #388